spec.bs

```diff
@@ -3643,7 +3680,7 @@ A <dfn>reporting context</dfn> is a [=struct=] with the following [=struct/items
 
 A <dfn>reporting context map</dfn> is a [=map=] from [=auction config=] to [=reporting context=].
 Here the keys are configurations for auctions that actually produce bids (e.g. not the top-level
-auction in a multi-party auction).
+auction in a multi-seller auction).

```

Comment:
Woops, actually this entire sentence is outdated, I changed it when adding P.Agg. stuff (since it made things messy for it and is basically irrelevant for forDebugOnly)

Reply:
turns out to be my branch was not updated. It's seller already after I rebased.

Reply:
Right, but I think the entire sentence should be removed since it's no longer correct (for reasons unrelated to this CL, I just noticed it based on your correction... and not when I should have removed it).

Reply:
oh sorry I misunderstood the UI and your comment... Removed the example sentence.